src/backend/src/prisma/schema.prisma

```diff
@@ -842,18 +842,20 @@ model Car {
 }
 
 model Organization {
-  organizationId String    @id @default(uuid())
-  name           String
-  dateCreated    DateTime  @default(now())
-  userCreatedId  String
-  userCreated    User      @relation(fields: [userCreatedId], references: [userId], name: "organizationCreator")
-  dateDeleted    DateTime?
-  userDeletedId  String?
-  userDeleted    User?     @relation(fields: [userDeletedId], references: [userId], name: "organizationDeleter")
-  treasurerId    String?
-  treasurer      User?     @relation(name: "treasurer", fields: [treasurerId], references: [userId])
-  advisor        User?     @relation(name: "advisor", fields: [advisorId], references: [userId])
-  advisorId      String?
+  organizationId            String    @id @default(uuid())
+  name                      String
+  dateCreated               DateTime  @default(now())
+  userCreatedId             String
+  userCreated               User      @relation(fields: [userCreatedId], references: [userId], name: "organizationCreator")
+  dateDeleted               DateTime?
+  userDeletedId             String?
+  userDeleted               User?     @relation(fields: [userDeletedId], references: [userId], name: "organizationDeleter")
+  treasurerId               String?
+  treasurer                 User?     @relation(name: "treasurer", fields: [treasurerId], references: [userId])
+  advisor                   User?     @relation(name: "advisor", fields: [advisorId], references: [userId])
+  advisorId                 String?
+  interestedinApplyingImage String?   @unique
```

Comment:
I dont love this name because we will have the application process eventually in finishline so the text will end up different but i think its ok for now unless you can think of something better